network-api/networkapi/wagtailcustomization/static/css/admin.css

```diff
@@ -53,3 +53,8 @@ fieldset ul li div label {
 #wagtail .wrapper input[type="checkbox"] {
   vertical-align: middle;
 }
+
+/* Displaying red asterisk for required fields. */
```

Comment:
I noticed that listblocks were not displaying the red asterisks for required fields in the cms, and found that in the wagtail css it was being set to "display:none" for some reason. 

Upon further research I found this PR for wagtail https://github.com/wagtail/wagtail/pull/6715, which mentions the same problem and a fix.

However, since we are not using the latest version of wagtail yet and the PR was pretty recent, I added this piece of CSS to make sure required fields display properly for the time being.